specification/versioning.md

```diff
@@ -77,3 +77,35 @@ until the end of that component’s existence:
   MUST be provided as the latest `PATCH` version for the latest `MINOR` version
   of the latest `MAJOR` and SHOULD NOT be provided for previous `PATCH` or
   `MINOR` releases.
+
+## Release life cycle management
+
+Each repository `MUST` use [GitHub releases mechanism](https://docs.github.com/en/repositories/releasing-projects-on-github/about-releases)
+to create a versioned snapshot of components and features
```

Comment:
```suggestion
to publish versioned artifacts of components and features
```

I'd like to avoid "snapshot" here because it has a very specific meaning, at least in Java.

Reply:
I updated my suggestion https://github.com/signalfx/gdi-specification/pull/225#discussion_r1159428244 to reflect your proposal